copilot/fastapi_proxy.py

```
@@ -0,0 +1,132 @@
+        target_url: The URL to which the request is proxied.
+        method: The HTTP method (e.g., "GET", "POST", "PUT", "DELETE") to use.
+        request: The incoming FastAPI request from which to extract the payload when applicable.
+        headers: Optional headers to include in the proxied request.
+        
+    Returns:
+        The JSON-decoded response from the target URL.
+        
+    Raises:
+        HTTPException: If an HTTP error occurs during the proxied request.
+    """
+    async with httpx.AsyncClient(verify=False) as client:
```

Ruff (0.8.2)

Probable use of `httpx` call with `verify=False` disabling SSL certificate checks

(S501)